### DISCUSSION TITLE
Outstanding Billing Problem

### DISCUSSION THREAD

**Discussion by andrealfarysie** [supabase/supabase#19673] [Questions] [answered]
Hi everyone, due to i can't get any respond from my ticket in email so i bring my issue in here, cause i dont know any contact that i can communicate with anymore in supabase.

I have problem with my outstanding bill. 
Firstly, i ever use Supabase for my project, and in that time i purchase a pro plan
[redacted]

I only use that instance for 3 days (event based usage), and never used it anymore.
Now i want to reactivate my instance, but problem with outstanding bill, when i check there are another 2 outstanding bill.

[redacted]
[redacted]

My question is, what outstanding bill that i need to pay to reactivate my instance, cause i dont think i use the instance in that time after i buy a pro plan for a 3 days usage.

even if its running, i dont think its exceed my free plan limit.
can Supabase team help me to check the outstanding bill that i mention above?

I really need to reactivated my instance in 10 hours ahead, even if i need to pay another $25. 
please help me.
thanks before

**Comment by kevcodez** [comment, marked as answer]
Hey @andrealfarysie, this is really not a place to discuss this as this easily exposes sensitive information publicly. I see you've created a ticket less than 3h ago. Note that we receive a large volume of tickets from free plan users and there's no guaranteed SLA for them.

I will reply on the ticket and remove the screenshots given they expose some sensitive information.

**Comment by andrealfarysie** [comment]
Hi Kevin, thanks for the heads up. Im really sorry for that discuss.
